api/spec/controllers/spree/api/users_controller_spec.rb

```diff
@@ -12,7 +12,7 @@ module Spree
       it "can get own details" do
         api_get :show, id: user.id, token: user.spree_api_key
 
-        json_response['email'].should eq user.email
+        expect(json_response['email']).to eq user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.